chainercv/extensions/evaluator/detection_coco_evaluator.py

```diff
@@ -0,0 +1,176 @@
+        ar/iou=0.50:0.95/area=large/max_dets=100/<label_names[l]>, \
+            [#coco_det_ext_2]_ [#coco_det_ext_5]_
+        map/iou=0.50:0.95/area=all/max_dets=100, \
+            [#coco_det_ext_3]_
+        map/iou=0.50/area=all/max_dets=100, \
+            [#coco_det_ext_3]_
+        map/iou=0.75/area=all/max_dets=100, \
+            [#coco_det_ext_3]_
+        map/iou=0.50:0.95/area=small/max_dets=100, \
+            [#coco_det_ext_3]_ [#coco_det_ext_5]_
+        map/iou=0.50:0.95/area=medium/max_dets=100, \
+            [#coco_det_ext_3]_ [#coco_det_ext_5]_
+        map/iou=0.50:0.95/area=large/max_dets=100, \
+            [#coco_det_ext_3]_ [#coco_det_ext_5]_
+        ar/iou=0.50:0.95/area=all/max_dets=1, \
+            [#coco_det_ext_4]_
+        ar/iou=0.50/area=all/max_dets=10, \
+            [#coco_det_ext_4]_
+        ar/iou=0.75/area=all/max_dets=100, \
+            [#coco_det_ext_4]_
+        ar/iou=0.50:0.95/area=small/max_dets=100, \
+            [#coco_det_ext_4]_ [#coco_det_ext_5]_
+        ar/iou=0.50:0.95/area=medium/max_dets=100, \
+            [#coco_det_ext_4]_ [#coco_det_ext_5]_
+        ar/iou=0.50:0.95/area=large/max_dets=100, \
+            [#coco_det_ext_4]_ [#coco_det_ext_5]_
+
+    .. [#coco_det_ext_1] Average precision for class \
+        :obj:`label_names[l]`, where :math:`l` is the index of the class. \
+        If class :math:`l` does not exist in either :obj:`pred_labels` or \
+        :obj:`gt_labels`, the corresponding value is set to :obj:`numpy.nan`.
+    .. [#coco_det_ext_2] Average recall for class \
+        :obj:`label_names[l]`, where :math:`l` is the index of the class. \
+        If class :math:`l` does not exist in either :obj:`pred_labels` or \
+        :obj:`gt_labels`, the corresponding value is set to :obj:`numpy.nan`.
+    .. [#coco_det_ext_3] The average of average precisions over classes.
+    .. [#coco_det_ext_4] The average of average recalls over classes.
+    .. [#coco_det_ext_5] Skip if :obj:`gt_areas` is :obj:`None`.
+
+    Args:
+        iterator (chainer.Iterator): An iterator. Each sample should be
+            following tuple :obj:`img, bbox, label, area, crowded`.
+        target (chainer.Link): A detection link. This link must have
+            :meth:`predict` method that takes a list of images and returns
+            :obj:`bboxes`, :obj:`labels` and :obj:`scores`.
+        label_names (iterable of strings): An iterable of names of classes.
+            If this value is specified, average precision and average
+            recalls for each class are reported.
+
+    """
+
+    trigger = 1, 'epoch'
+    default_name = 'validation'
+    priority = chainer.training.PRIORITY_WRITER
+
+    def __init__(
+            self, iterator, target,
+            label_names=None):
+        if not _available:
+            raise ValueError(
+                'Please install pycocotools \n'
+                'pip install -e \'git+https://github.com/pdollar/coco.git'
```

Comment:
Why are you using pdollar's repository? https://github.com/cocodataset/cocoapi looks official one. I know `eval_detection_coco` is also using it.